presto-benchto-benchmarks/src/main/resources/benchmarks/presto/tpcds.yaml

```diff
@@ -5,29 +5,45 @@ prewarm-runs: 2
 before-execution: sleep-4s, presto/session_set_reorder_joins.sql
```

Comment:
do not mention time in the commit message, time is in function of hardware used. Or maybe describe the hardware and Presto configuraiton a bit in the commit message.

Reply:
`in a reasonable timeframe` I would change to `in a comparable timeframe`

Reply:
but it is a time categorisation and surely it is a hardware function, and actually it's probably not linear

anyway I could add a general high level note on the hardware

but it's not because we want it to be comparable, times under 10s are also comparable, but they're not reasonable in the sense that they contains too much noise, and times over 60s are just making things run too long, ergo I think the adjective "reasonable" it good enough